docs/parameters.rst

```diff
@@ -2,3 +2,276 @@
+However, the adapters have to be careful to ensure that they don't double report by e.g. providing ``Radiative Forcing|Aerosols|Direct Effect`` and ``Radiative Forcing|Aerosols|NOx``.
+
+- ``Radiative Forcing`` (W/m\*\*2)
+- ``Radiative Forcing|Aerosols`` (W/m\*\*2)
+- ``Radiative Forcing|Aerosols|Direct Effect`` (W/m\*\*2)
+- ``Radiative Forcing|Aerosols|Indirect Effect`` (W/m\*\*2)
+- ``Radiative Forcing|Aerosols|SOx`` (W/m\*\*2)
+- ``Radiative Forcing|Aerosols|NOx`` (W/m\*\*2)
+- ``Radiative Forcing|Aerosols|OC`` (W/m\*\*2)
+- ``Radiative Forcing|Aerosols|BC`` (W/m\*\*2)
+- ``Radiative Forcing|Land-use Change`` (W/m\*\*2)
+- ``Radiative Forcing|Black Carbon on Snow`` (W/m\*\*2)
+- ``Radiative Forcing|Volcanic`` (W/m\*\*2)
+- ``Radiative Forcing|Solar`` (W/m\*\*2)
+- ``Radiative Forcing|External`` (W/m\*\*2)
+- ``Radiative Forcing|CO2`` (W/m\*\*2)
+- ``Radiative Forcing|CH4`` (W/m\*\*2)
+- ``Radiative Forcing|N2O`` (W/m\*\*2)
+- ``Radiative Forcing|NF3`` (W/m\*\*2)
+- ``Radiative Forcing|CF4`` (W/m\*\*2)
+- ``Radiative Forcing|C2F6`` (W/m\*\*2)
+- ``Radiative Forcing|C3F8`` (W/m\*\*2)
+- ``Radiative Forcing|cC4F8`` (W/m\*\*2)
+- ``Radiative Forcing|C4F10`` (W/m\*\*2)
+- ``Radiative Forcing|C5F12`` (W/m\*\*2)
+- ``Radiative Forcing|C6F14`` (W/m\*\*2)
+- ``Radiative Forcing|C7F16`` (W/m\*\*2)
+- ``Radiative Forcing|C8F18`` (W/m\*\*2)
+- ``Radiative Forcing|CCl4`` (W/m\*\*2)
+- ``Radiative Forcing|CHCl3`` (W/m\*\*2)
+- ``Radiative Forcing|CH2Cl2`` (W/m\*\*2)
+- ``Radiative Forcing|CH3CCl3`` (W/m\*\*2)
+- ``Radiative Forcing|CH3Cl`` (W/m\*\*2)
+- ``Radiative Forcing|CH3Br`` (W/m\*\*2)
+- ``Radiative Forcing|HFC23`` (W/m\*\*2)
+- ``Radiative Forcing|HFC32`` (W/m\*\*2)
+- ``Radiative Forcing|HFC4310`` (W/m\*\*2)
+- ``Radiative Forcing|HFC125`` (W/m\*\*2)
+- ``Radiative Forcing|HFC134a`` (W/m\*\*2)
+- ``Radiative Forcing|HFC143a`` (W/m\*\*2)
+- ``Radiative Forcing|HFC152a`` (W/m\*\*2)
+- ``Radiative Forcing|HFC227ea`` (W/m\*\*2)
+- ``Radiative Forcing|HFC236fa`` (W/m\*\*2)
+- ``Radiative Forcing|HFC245fa`` (W/m\*\*2)
+- ``Radiative Forcing|HFC365mfc`` (W/m\*\*2)
+- ``Radiative Forcing|CFC11`` (W/m\*\*2)
+- ``Radiative Forcing|CFC12`` (W/m\*\*2)
+- ``Radiative Forcing|CFC113`` (W/m\*\*2)
+- ``Radiative Forcing|CFC114`` (W/m\*\*2)
+- ``Radiative Forcing|CFC115`` (W/m\*\*2)
+- ``Radiative Forcing|HCFC22`` (W/m\*\*2)
+- ``Radiative Forcing|HCFC141b`` (W/m\*\*2)
+- ``Radiative Forcing|HCFC142b`` (W/m\*\*2)
+- ``Radiative Forcing|SF6`` (W/m\*\*2)
+- ``Radiative Forcing|SO2F2`` (W/m\*\*2)
+- ``Radiative Forcing|Halon1202`` (W/m\*\*2)
+- ``Radiative Forcing|Halon1211`` (W/m\*\*2)
+- ``Radiative Forcing|Halon1301`` (W/m\*\*2)
+- ``Radiative Forcing|Halon2402`` (W/m\*\*2)
+
+
+Material Fluxes
+***************
+
+These variables can be used to store the flux of material within the model.
+They should be of the form ``X to Y Flux`` where the material is flowing from ``X`` into ``Y`` (and hence negative values represent flows from ``Y`` into ``X``).
```

Comment:
Maybe ``("Flux", "X", "Y")``?

Reply:
Would be super confusing given the convention about variables being the sum of all variables one level below I think. I'm not convinced my solution is perfect either though so if we can think of something smarter that'd be ideal.

Reply:
@swillner I would propose sticking with my solution, I don't have any better ideas.